front/yarn.lock

```diff
@@ -2529,7 +2529,7 @@ fsevents@^1.2.7:
     bindings "^1.5.0"
     nan "^2.12.1"
 
-fsevents@~2.3.1, fsevents@~2.3.2:
+fsevents@^2.0.6, fsevents@~2.3.1, fsevents@~2.3.2:
```

Comment:
I don't really understand this change, why was ^2.0.6 added here? :)

Reply:
yarn.lock file is auto-generated by running `yarn install`.
TLDR;  https://classic.yarnpkg.com/en/docs/yarn-lock/
actually you should not pay much attention what is in there